crates/iota/src/validator_commands.rs

```diff
@@ -556,7 +556,9 @@ impl IotaValidatorCommand {
                     .await?;
                 let active_validators = match iota_system_state {
                     IotaSystemStateSummary::V1(v1) => v1.active_validators,
-                    IotaSystemStateSummary::V2(v2) => v2.active_validators,
+                    IotaSystemStateSummary::V2(v2) => {
+                        v2.iter_committee_members().cloned().collect::<Vec<_>>()
```

Comment:
You can use `to_committee_members`

Reply:
We can avoid match entirely. Fixed in the new commit.